apps/omg_watcher/lib/omg_watcher/api/transaction.ex

```diff
@@ -46,7 +46,7 @@ defmodule OMG.Watcher.API.Transaction do
   Length of the list is limited by `limit` argument
   """
   @decorate measure_event()
-  @spec get_transactions(Keyword.t()) :: list(%DB.Transaction{})
+  @spec get_transactions(Keyword.t()) :: Paginator.t()
```

Comment:
I made this change to satisfy the dialyzer. It looks like `DB.Transaction.get_by_filters()` actually does return a `Paginator.t()`, so the change made sense. 🤷‍♂